corehq/util/model_log.py

```diff
@@ -26,11 +29,19 @@ def log_model_change(user, model_object, message=None, fields_changed=None):
 
         message = {'changed': {'fields': fields_changed}}
 
+    content_type_id = None
+    model_object_pk = None
+    if isinstance(model_object, Model):
+        content_type_id = get_content_type_for_model(model_object).pk
+        model_object_pk = model_object.pk
+    elif isinstance(model_object, Document):
+        model_object_pk = model_object.get_id
```

Comment:
I'm not opposed to this change for other types but since every couch user has an associated Django user I think this should always be called with the Django user for both the 'user' and the 'model_object'

Reply:
From HQ perspective I have always thought of the couch user as the "user". Even externally Web Users and Mobile Users are a thing but django user is not. Most of the code base also mostly refers to couch user. Hence the implementation to note things as couch user.
Django user should be okay too, Updated to use django user.

Reply:
https://github.com/dimagi/commcare-hq/pull/27755/commits/e4586112d4e05186441671ebe1ad1ff5bb6a9448